src/esp/assets/GenericSemanticMeshData.cpp

```diff
@@ -61,11 +61,11 @@ GenericSemanticMeshData::buildSemanticMeshData(
 
   if (semanticFilename.find(".ply") != std::string::npos) {
     // Generic Semantic PLY meshes have -Z gravity
-    const quatf T_esp_scene =
-        quatf::FromTwoVectors(-vec3f::UnitZ(), geo::ESP_GRAVITY);
+    const Mn::Quaternion T_esp_scene = Mn::Quaternion{
+        quatf::FromTwoVectors(-vec3f::UnitZ(), geo::ESP_GRAVITY)};
 
     for (auto& xyz : semanticData->cpu_vbo_) {
-      xyz = T_esp_scene * xyz;
+      xyz = T_esp_scene.toMatrix() * xyz;
```

Comment:
Should the toMatrix() here be cached to help with vectorization?

Reply:
thanks, yeah that's ugly.